File: 2 player/win.py
import sys

import play1
import menu1

def winner(b):
  w = 0
  bl = 0
  for i in range(9):
    for j in range(9):
      if b[i][j][1] == 'o':
        w += 1
      elif b[i][j][1] == 'x':
        bl += 1
  print(f"POINTS\nBlack - {bl}\nWhite - {w}")
  if w > bl:
    print("White is the winner")
  elif bl > w:
    print("Black is the winner")
  elif w == bl:
    print("Its a tie")
  print("Enter   1.To replay\n\t2.to go back to main menu")
  while True:
    try:
      y = int(input())
      if y == 1:
        play1.game()
        break
      elif y == 2:
        menu1.menu()
        break
      else:
        print("Enter a valid number")
    except ValueError:
      print("Invalid entry.Enter a valid integer value")
      sys.exit(0)

File: games_java/win.java
import java.io.*;
class win
{
    public static void winner(String b[][])throws Exception
    {
        int w=0;
        int bl=0;
        for(int i=1;i<9;i++)
        {
            for(int j=1;j<9;j++)
            {
                if(b[i][j].charAt(1)=='o')
                {
                    w++;
                }
                else if(b[i][j].charAt(1)=='x')
                {
                    bl++;
                }
            }
        }
        System.out.println("POINTS\nBlack - "+bl+"\nWhite - "+w);
        if(w>bl)
        {
            System.out.println("White is the winner");
        }
        else if(bl>w)
        {
            System.out.println("Black is the winner");
        }
        else if(w==bl)
        {
            System.out.println("Its a tie");
        }
        BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Enter   1.To replay\n\t2.to go back to main menu");
        int y=0;
        do
        {
            try
            {
               y=Integer.parseInt(br.readLine());
            }
            catch (Exception e1)
            {
                System.out.println("Invalid entry.Enter a valid integer value");
                continue;
            }
        }
        while(y!=1&&y!=2);
        switch(y)
        {
            case 1:play1.game();
            break;
            case 2:menu1.menu();
            break;
        }
    }
}